docs/cunumeric/source/user/notebooks/stencil.ipynb

```diff
@@ -0,0 +1,221 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "14c01af2-d331-4aa4-9639-41353601a767",
+   "metadata": {},
+   "source": [
+    "# Jacobi stencil\n",
+    "\n",
+    "<details>\n",
+    "<summary>License</summary>\n",
+    "<pre>\n",
+    "\n",
+    "Copyright 2023 NVIDIA Corporation\n",
+    "\n",
+    "Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "you may not use this file except in compliance with the License.\n",
+    "You may obtain a copy of the License at\n",
+    "\n",
+    "     http://www.apache.org/licenses/LICENSE-2.0\n",
+    "\n",
+    "Unless required by applicable law or agreed to in writing, software\n",
+    "distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "See the License for the specific language governing permissions and\n",
+    "limitations under the License.\n",
+    "</pre>\n",
+    "</details>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "35c48e6f-1bde-4aac-af55-b7218cc22491",
+   "metadata": {},
+   "source": [
+    "To get started, `import cunumeric as np` (just the same way we would import `numpy`)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "bfc50224-f762-4ec0-8ded-671cbd1df559",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import cunumeric as np  # instead of numpy"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "19f09f10-a627-4ca2-bf7c-a75c71f54ac5",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from legate.timing import time\n",
+    "\n",
+    "def jacobi(grid, I): \n",
+    "    print(\"Running Jacobi stencil... \", end=\"\")\n",
+    "    \n",
+    "    center = grid[1:-1, 1:-1]\n",
+    "    north = grid[0:-2, 1:-1]\n",
+    "    east = grid[1:-1, 2:]\n",
+    "    west = grid[1:-1, 0:-2]\n",
+    "    south = grid[2:, 1:-1]\n",
+    "\n",
+    "    start = time()\n",
+    "    for i in range(I):\n",
+    "        average = center + north + east + west + south\n",
+    "        center[:] = 0.2 * average\n",
+    "    stop = time()\n",
+    "\n",
+    "    print(f\"Elapsed Time: {(stop - start)/1000} ms\")\n",
```

Comment:
Same comment as before. Timings would be a lot more interesting if shown for both CPU and multi-GPU